proto/dydxprotocol/vault/params.proto

```diff
@@ -25,4 +27,14 @@ message Params {
 
   // The duration that a vault's orders are valid for.
   uint32 order_expiration_seconds = 6;
+
+  // The number of quote quantums in quote asset that a vault with no perpetual
+  // positions must have to activate, i.e. if a vault has no perpetual positions
+  // and has strictly less than this amount of quote asset, it will not
+  // activate.
+  bytes activation_threshold_quote_quantums = 7 [
```

Comment:
Nit: consider using uint64 why is sufficient for all realistic threshold here (`uint64` max = 18 trillion USDC)

Reply:
i agreed that uint64 is probably sufficient but a deposit to vault is represented as a serializable int already and wanted to keep it consistent